pages/common/git-squash.md

```diff
@@ -0,0 +1,21 @@
+# git squash
+
+> Squash N last changes up to a ref'ed commit into current HEAD.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-squash>.
+
+- Squash everything from a specific branch into the current branch:
+
+`git squash {{source-branch}}`
+
+- Squash everything from a commit reference into the current branch:
+
+`git squash {{commit-ref}}`
```

Comment:
```suggestion
`git squash {{commit-hash}}`
```

Reply:
I saw other places using the `commit-ref` term, would this change break consistency?

Reply:
We use `commit|branch_name|ref` in a page, it's not the same, `ref` relates to a tag.
We use just `commit` for commit sha values.
```suggestion
`git squash {{commit}}`
```